usage/installer/README.md

```diff
@@ -0,0 +1,48 @@
+# SeaClouds
+
+## Deploy SeaClouds
+A deployment of SeaClouds can be launched using Apache Brooklyn. We currently support deployments against Bring Your 
+Own Nodes (BYON) and to all the IaaS provider supported by [Apache jclouds](http://jclouds.org).
+
+# Deploy SeaClouds platform
+
+## Launching SeaClouds on the cloud
+
+```bash
+cd installer/target/seaclouds-installer-dist/seaclouds-installer
+nohup ./start.sh --app blueprints/seaclouds.yaml --location <location> &
+```
+
+## Launching SeaClouds on BYON
+
+Make sure you have [Vagrant](https://www.vagrantup.com/), [Virtual Box](https://www.virtualbox.org/)
+
+```bash
+cd usage/installer/target/seaclouds-installer-dist/seaclouds-installer
+pushd .
+cd byon
+vagrant up 
+popd
+nohup ./start.sh &
+tail -f nohup.out
+```
+This spins up a virtual environment, made up of 2 VMs, that is accessible at `192.168.100.10` and `192.168.100.11`.
+
+Finally, copy and paste [seaclouds blueprint](seaclouds-on-byon.yaml) to deploy the SeaClouds platform on the 2 VMs.
```

Comment:
This link seems broken. Maybe:
`./src/main/assembly/files/blueprints/seaclouds-on-byon.yaml`